src/host/ut_host/AliasTests.cpp

```diff
@@ -181,258 +180,4 @@ class AliasTests
-
-        auto tokensActual = Alias::s_Tokenize(tokenStr);
-
-        VERIFY_ARE_EQUAL(tokensExpected.size(), tokensActual.size());
-
-        for (size_t i = 0; i < tokensExpected.size(); i++)
-        {
-            VERIFY_ARE_EQUAL(String(tokensExpected[i].data()), String(tokensActual[i].data()));
-        }
-    }
-
-    TEST_METHOD(GetArgString)
-    {
-        BEGIN_TEST_METHOD_PROPERTIES()
-            TEST_METHOD_PROPERTY(L"Data:targetExpectedPair",
-                                 L"{"
-                                 L"alias arg1 arg2 arg3=arg1 arg2 arg3,"
-                                 L"aliasOnly="
-                                 L"}")
-        END_TEST_METHOD_PROPERTIES()
-
-        std::wstring target;
-        std::wstring expected;
-        _RetrieveTargetExpectedPair(target, expected);
-
-        auto actual = Alias::s_GetArgString(target);
-
-        VERIFY_ARE_EQUAL(String(expected.data()), String(actual.data()));
-    }
-
-    TEST_METHOD(NumberedArgMacro)
-    {
-        BEGIN_TEST_METHOD_PROPERTIES()
-            TEST_METHOD_PROPERTY(L"Data:targetExpectedPair",
-                                 L"{"
-                                 L"1=one,"
-                                 L"2=two,"
-                                 L"3=three,"
-                                 L"4=four,"
-                                 L"5=five,"
-                                 L"6=six,"
-                                 L"7=seven,"
-                                 L"8=eight,"
-                                 L"9=nine,"
-                                 L"A=,"
-                                 L"0=,"
-                                 L"}")
-        END_TEST_METHOD_PROPERTIES()
-
-        std::wstring target;
-        std::wstring expected;
-        _RetrieveTargetExpectedPair(target, expected);
-
-        std::deque<std::wstring> tokens;
-        tokens.emplace_back(L"alias");
-        tokens.emplace_back(L"one");
-        tokens.emplace_back(L"two");
-        tokens.emplace_back(L"three");
-        tokens.emplace_back(L"four");
-        tokens.emplace_back(L"five");
-        tokens.emplace_back(L"six");
-        tokens.emplace_back(L"seven");
-        tokens.emplace_back(L"eight");
-        tokens.emplace_back(L"nine");
-        tokens.emplace_back(L"ten");
-
-        // if we expect non-empty results, then we should get a bool back saying it was processed
-        const auto returnExpected = !expected.empty();
-
-        std::wstring actual;
-        const auto returnActual = Alias::s_TryReplaceNumberedArgMacro(target[0], actual, tokens);
-
-        VERIFY_ARE_EQUAL(returnExpected, returnActual);
-        VERIFY_ARE_EQUAL(String(expected.data()), String(actual.data()));
-    }
-
-    TEST_METHOD(WildcardArgMacro)
-    {
-        BEGIN_TEST_METHOD_PROPERTIES()
-            TEST_METHOD_PROPERTY(L"Data:targetExpectedPair",
-                                 L"{"
-                                 L"*=one two three,"

```

Comment:
This is covered by `L"bar $*=bar one two three four five six seven eight nine ten eleven twelve%,"` above.